packages/experimental/RadioGroup/src/Radio/useRadio.win32.ts

```diff
@@ -51,10 +52,59 @@ export const useRadio = (props: RadioProps): RadioInfo => {
     }
   }, []);
 
+  const isCircularNavigation = true;
+
+  const onInvoke = React.useCallback(
+    (e: KeyPressEvent) => {
+      if (
+        e.nativeEvent.key == 'ArrowDown' ||
+        e.nativeEvent.key == 'ArrowRight' ||
+        e.nativeEvent.key == 'ArrowUp' ||
+        e.nativeEvent.key == 'ArrowLeft'
+      ) {
```

Comment:
Is this if statement necessary? I would just do `if (e.nativeEvent.key === 'ArrowDown' || e.nativeEvent.key === 'ArrowRight')` and `else if (e.nativeEvent.key === 'ArrowUp' || e.nativeEvent.key === 'ArrowLeft')` down below.

Reply:
Thanks for the suggestion, simplified!